docs/New-Maintainer-Checklist.md

```diff
@@ -58,7 +58,7 @@ If they accept, follow a few steps to get them set up:
 - Add them to the [Jenkins' GitHub Authorization Settings admin user names](https://jenkins.brew.sh/configureSecurity/) so they can adjust settings and restart jobs.
 - Add them to the [Jenkins' GitHub Pull Request Builder admin list](https://jenkins.brew.sh/configure) to enable `@BrewTestBot test this please` for them.
 - Invite them to the [`homebrew-maintainers` private maintainers mailing list](https://lists.sfconservancy.org/mailman/admin/homebrew-maintainers/members/add).
-- Invite them to the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin/invites) (and ensure they've read the [communication guidelines](Maintainer-Guidelines.md#communication)).
+- Invite them to the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin/invites) (and ensure they've read the [communication guidelines](Maintainer-Guidelines.md#communication)) and ask them to use their real name there (rather than a pseudonym they may use on e.g. GitHub).
```

Comment:
Does "real name" include shortenings? For example I'm "Issy" - only my family call me my full name "Isabell".

Reply:
@issyl0 I'm thinking "real" as opposed to "legal" here. Would love any wording suggestions on how to better communicate that 🤔

Reply:
I think this is fine, I just wanted to clarify.